### PR TITLE
fix tree detail height

### LINKED ISSUE #58: Set height of components dynamic based on different devices
I am struggling with iPad, for example.
Is mainly about `InventoryListPage` and `TreeDetails`. 
The position of the popovers for iOS is also not ideal.

### PR DESCRIPTION
**Pull request by mmaelicke**
fixes #58 

I tested for Desktop, Mobile and iPad Air (in Chrome) to fix the width of the Tree Detail card.

I am not sure, what exactly the problem is on mobile devices. I will add a commented review to explain my changes.

### CODE REVIEW

File: src/pages/DesktopPage.tsx [hunk 1]
```diff
@@ -83,11 +83,13 @@ const DesktopPage: React.FC = () => {
                     <IonCard
                       style={{
                         position: "absolute",
+                        margin: "0px",
```
Comment (mmaelicke) [inline src/pages/DesktopPage.tsx:86]: The card has a margin, that was screwing up on iPads due to different values in relation to current viewport size

File: src/pages/DesktopPage.tsx [hunk 2]
```diff
@@ -83,11 +83,13 @@ const DesktopPage: React.FC = () => {
...
                         top: 60,
-                        left: 12,
+                        left: 20,
```
Comment (mmaelicke) [inline src/pages/DesktopPage.tsx:88]: This is the absolute displacement of the Buttons above the details

File: src/pages/DesktopPage.tsx [hunk 3]
```diff
@@ -83,11 +83,13 @@ const DesktopPage: React.FC = () => {
...
                         // width: "33vw",
                         width: "400px",
-                        height: "90%",
+                        maxWidth: "80%",
+                        height: "calc(100% - 60px - 20px)",
```
Comment (mmaelicke) [inline src/pages/DesktopPage.tsx:92]: the height is now calculated for the current height minus the fixed offsets

File: src/pages/DesktopPage.tsx [hunk 4]
```diff
@@ -83,11 +83,13 @@ const DesktopPage: React.FC = () => {
...
                       style={{
                         position: "absolute",
+                        margin: "0px",
                         top: 60,
-                        left: 12,
+                        left: 20,
                         // width: "33vw",
                         width: "400px",
-                        height: "90%",
+                        maxWidth: "80%",
```
Comment (mmaelicke) [inline src/pages/DesktopPage.tsx:91]: Makes the card slightly smaller on iPad to not overlay the position button on the map

**Comment by JesJehle** [comment]
For iPad, the card is still too high (Brave)
image

**Comment by mmaelicke** [comment]
> For iPad, the card is still too high (Brave)
> 

Can you also try chrome or safari? On both, a iPad screen size works for me, or did you try an actual iPad?

**Review by JesJehle — APPROVED** [review]
It's working now. No idea why ...

**Pull request merged**